src/Helpers.php

```diff
@@ -36,9 +36,7 @@ function default_video_sources()
 
     function cl_upload_url($options = array())
     {
-        if (!@$options["resource_type"]) {
-            $options["resource_type"] = "auto";
-        }
+        $options['resource_type'] = isset($options['resource_type']) ? $options['resource_type'] : 'auto';
```

Comment:
It would be neater to use  `Cloudinary::option_get()` function here, like this:
```suggestion
         $options['resource_type'] = Cloudinary::option_get($options, 'resource_type', 'auto');
```